db/seeds.rb

```diff
@@ -1,50 +1,57 @@
+# frozen_string_literal: true
+
 require 'csv'
 require 'faker'
 
 # add cities in DB
 
-City.all.destroy_all
+City.find_each(&:destroy!)
 
-source_file = "/Users/eleonore/Documents/Code/enm-city-allocator/db/enm_cities_seats.csv"
+source_file = '/Users/eleonore/Documents/Code/enm-city-allocator/db/jurisdictions_and_choices.csv'
 
 CSV.foreach(source_file, headers: true) do |row|
-	City.create(
-		name: row[1],
-		nb_seats: row[2]
-		)
+  City.create!(
+    name: row[0],
+    nb_seats: row[1],
+    wishes: [row[2], row[3], row[4], row[5], row[6], row[7]].map(&:to_i)
```

Comment:
```suggestion
    wishes: row[2..7].map(&:to_i)
```

Reply:
@Berardpi Totally agree with you, that's what I tried to do in the first place. But I wanted/needed, on this other branch 'allocator-script', to start from the latest version of the code (on the models branch, not approved/merged) and this branch originated from master (so quite a few commits back). I think I found the solution yesterday (but after having pushed the latests commits on models): 
$ git branch -D allocator script
$ git pull origin master
$  git checkout -b allocator script
Was it the right way to do it? I'm guessing there is another way with more advanced Git commands (rebase?) but I'm not familiar with them yet and was kind of in a hurry as I have to finish working on the allocator algo by this week-end.